ext/opentelemetry-ext-azure-monitor/src/opentelemetry/ext/azure_monitor/trace.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry.sdk.trace.export import SpanExporter, SpanExportResult
+
+
+class AzureMonitorSpanExporter(SpanExporter):
+    def __init__(self):
+        pass
+
+    def export(self, spans):
+        for span in spans:
+            print(span)  # TODO: add actual implementation here
```

Comment:
mmm, maybe we want to extend  `__repr__`, implement `__str__` or something similar to get a pretty print of the stamp.

Reply:
Yes we should.

Reply:
Done.